pkg/controllers/scheduling/schedule.go

```diff
@@ -48,6 +49,7 @@ type FilterResult struct {
 
 type PriorizeResult struct {
```

Comment:
Please also help to correct some typos in schedule.go:
- PriorizeResult --> PrioritizerResult
- PrioritizeSore --> PrioritizerScore
- PriorizeResults() --> PrioritizerResults()

Reply:
fixed.